pytket/extensions/cutensornet/mps/mps.py

```diff
@@ -85,76 +85,68 @@ def __exit__(self, exc_type: Any, exc_value: Any, exc_tb: Any) -> None:
         self._is_destroyed = True
 
 
-class MPS:
-    """Represents a state as a Matrix Product State.
-
-    Attributes:
-        chi (int): The maximum allowed dimension of a virtual bond.
-        truncation_fidelity (float): The target fidelity of SVD truncation.
-        tensors (list[Tensor]): A list of tensors in the MPS; ``tensors[0]`` is
-            the leftmost and ``tensors[len(self)-1]`` is the rightmost; ``tensors[i]``
-            and ``tensors[i+1]`` are connected in the MPS via a bond. All of the
-            tensors are rank three, with the dimensions listed in ``.shape`` matching
-            the left, right and physical bonds, in that order.
-        canonical_form (dict[int, Optional[DirectionMPS]]): A dictionary mapping
-            positions to the canonical form direction of the corresponding tensor,
-            or ``None`` if it the tensor is not canonicalised.
-        qubit_position (dict[pytket.circuit.Qubit, int]): A dictionary mapping circuit
-            qubits to the position its tensor is at in the MPS.
-        fidelity (float): A lower bound of the fidelity, obtained by multiplying
-            the fidelities after each contraction. The fidelity of a contraction
-            corresponds to ``|<psi|phi>|^2`` where ``|psi>`` and ``|phi>`` are the
-            states before and after truncation (assuming both are normalised).
-    """
+class ConfigMPS:
+    """Configuration class for simulation using MPS."""
 
     def __init__(
         self,
-        libhandle: CuTensorNetHandle,
-        qubits: list[Qubit],
         chi: Optional[int] = None,
         truncation_fidelity: Optional[float] = None,
-        float_precision: Optional[Union[np.float32, np.float64]] = None,
+        k: int = 4,
+        optim_delta: float = 1e-5,

```

Comment:
This parameter is not currently being used below.

Reply:
There is this line however:
`self.optim_delta = 1e-5`

Reply:
Good catch, thanks! Updated.